index.html

```diff
@@ -66,6 +66,199 @@ <h3>I'm a Software Developer</h3>
+        <span class="see-project">See Project</span>
+      </div>
+      <div class="projects-done">
+        <div class="project-name">
+          <p class="name"> Professional Art</p>
+          <p class="name">Printing Data</p>
+        </div>  
+        <p class="project-note">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+        <div class="project-stack">
+            <ul class="project-stack-used">
+                <li class="tech-used">HTML</li>
+                <li class="tech-used">Bootstrap</li>
+                <li class="tech-used">Ruby</li>
+            </ul>
+        </div>
+        <span class="see-project">See Project</span>
+      </div>
+        
+       
+      </div>
+     
+    </section>
+    <section class="about-myself">
+      <div class="about-me">
+        <h1 class="about-me-heading">About me</h1>
+        <p class="about-me-note">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        <span class="get-my-resume">Get my resume</span>
+      </div>
+      <hr class="stack">
+      <ul class="stacks">
+          <li class="languages">Languages</li>
+            <ul class="list">
+             <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          <li class="frameworks">Frameworks</li>
+            <ul class="list">
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>Rspec</li>
+            <li>Capybara</li>
+            </ul>
+          <li class="skills">Skills</li>
+            <ul class="list">
+              <li>Database Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+             <li>API Design</li>
+              <li>Web Development</li>
+            </ul>
+        </ul>
+        <!-- <div class="form">
+           <p class="form-note">
+            I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+          </p>
+          <form action="submit" class="inputs">
+            <input type="text" placeholder="Full Name">
+            <input type="email" placeholder="Email Address">
+            <textarea placeholder="Write me something..." cols="10" rows="10"></textarea>
+        </form>
+        <input class='get-in-touch' type="button" value="Get in touch">
+        </div>
+        <footer>
+          <ul class="footer-icons">
+          <li>
+            <a href="#"
+              ><img src="./icons/icon - git.svg" alt="github-icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="./icons/Icon - Linkedin.svg" alt="Linkedin-icon"
+            /></a>
+          </li>
+          <li>
+            <a href="#"
```

Comment:
- [ ] I would suggest you to remove commented out code form your code base. Commented-out code is distracting and confusing for developers who read the surrounding code, and its significance is often unclear. It will not get compiled or tested when the code around it changes, so it's likely to break over time. For these reasons, commented-out code should be avoided.